<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<!-- Created by: Clare Bates Congdon, July 11, 2018  -->
<head>
  <title>Clare Bates Congdon</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <link rel="stylesheet" type="text/css" href="style.css">
</head>

<body>

<div class="topnav"><small>
  <a href="index.html">Home</a>
  <a href="cv.html">Brief CV</a>
  <a class="active" href="projects.html">Projects</a>
  <a href="publications.html">Publications</a>
  <a href="teaching.html">Teaching</a>
  <a href="contact.html">Contact</a>
</small>
</div>
<br>

<h3>Some Recent Projects -- Bioinformatics</h3><br>
<table cellpadding=2>
<tr valign=top><td>
    <img src="Graphics/motifExampleSmall.jpg" align=left width=100 height=50 
    border=1 hspace=0 alt=""> 
    </td>
<td>
     <b>GAMI: genetic algorithms for motif inference</b><br>
     <small>
     Synopsis: In this project, we develop computational tools to infer
     regulatory regions in noncoding DNA, based on conservation over
     evolutionary time, and including biochemical data from
     <a href="https://www.encodeproject.org/">the ENCODE project</a>.<br>
     Current and recent collaborators: Leif Oxburgh, Carol Kim, Jim Coffman
     <br>
     Current staff and students: Craig Lessard, Sam McFarland, Brenna Gardner,
     Naomi Heuer, Amanda Castonguay, Ben Congdon-Jones
     </small>
     <ul>
       <li> With Carol Kim's lab at the University of Maine, we have
       computationally identified regulatory elements for CFTR, the gene
       responsible for cystic fibrosis, and completed preliminary biological
       validations of these.
       </li>
       <li> With Jim Coffman's lab at the Mount Desert Island Biological Lab,
       we have computationally identified regulatory regions for FOXJ1, a
       gene important in cilia development, and completed preliminary
       biological validations of these.
       </li>
       <li> With Leif Oxburgh's lab at the Maine Medical Center Research
       Institute, we have computationally identified regulatory regions for
       PDGFRb, a gene critical to kidney development, and completed
       preliminary biological validations of these.
       </li>
     </ul>
     <br>
</td></tr>

<tr valign=top><td>
    <img src="Graphics/gps.icon.jpg" align=left width=100
    border=1 hspace=0 alt=""> 
  </td>

  <td> <b>GPS: Genome Pattern Scan</b><br>
     <small>
     Synopsis: In this project, we develop computational tools that identify
     genes that might be influenced by specific regulatory factors (specified
     by the user).<br>
     Collaborators: Leif Oxburgh, Mary Ann Asson-Batres
     <br>
     Staff and students: Sam McFarland, Craig Lessard, Jen Fetting
     </small>
     <ul> 
       <li>With Leif Oxburgh's lab at the Maine Medical Center Research
        Institute, we are working to identify genes that might be regulated by
        FOXD1.
       </li>
       <li> With Mary Ann Asson-Batres of Tennessee State University and Doug
       Sawyer of the Maine Medical Center Research Institute, we are working
       to identify genes that might be regulated by the presence of Vitamin A.
       </li>
     </ul>
     <br>
</td></tr>

<tr><td>
    <img src="Graphics/cbcTreeUnrootedSmall.jpg" align=left width=100 height=40
    border=1 hspace=0 alt=""> 
    </td>
<td>
     <b>Gaphyl: genetic algorithms for phylogenetics</b><br>
     <small>
     Synopsis: Find the most plausible evolutionary relationships among
     species<br> 
     Staff and students: Emily Greenfest, Josh Ladieu, Kevin
     Septor, Lunn Sawyer
     </small>

</td></tr>
</table>
<br><br>

<h3>Some Recent Projects -- AI and Games</h3><br>
<table cellpadding=2>

<tr valign=top><td>
    <img src="Graphics/mario.jpg" align=right width=100
    border=1 hspace=0 alt=""> 
    </td>
<td>
   <b>REALM: A Rule-Based Evolutionary Computation Agent that Learns to Play
   Mario</b>
   <br> Winner of the 2010 Mario Learning Track and the 2010 Mario
   Gameplay Track at the Computational Intelligence and Games Conference,
   Copenhagen Denmark, August 2010
   <br> <a href="Teaching/Info/mario.html"> Videos of the agent</a>
     <small>
       <br>
       Synopsis: Using an evolutionary computation approach to learn to play
       Mario. <br>
     Masters project by Slawomir Bojarski<br><br>
     </small>
</td></tr>

<tr valign=top><td>
    <img src="Graphics/UnrealSquareSmall.jpg" align=right width=100
    border=1 hspace=0 alt=""> 
    </td>
<td>
  <b>Learning to Play Unreal</b><br>
  Winner of the 2009 Unreal competition at the Congress on Evolutionary
  Computation, Trondheim Norway, May 2009<br>
     <small>
     Synposis: Development of Chimera, an evolutionary approach to learning
    rule-based systems and Agent Smith, an agent to play Unreal. A hand-coded
    version of Agent Smith won the <a
    href="http://www.cec-2009.org/competitions.shtml#unreal">CEC 2009 Unreal
    Tournament competition</a>.<br>
     Independent study project by Ryan Small<br><br><br>
     </small>
</td></tr>

<tr valign=top><td>
    <img src="Graphics/mpmlevel4.jpg" align=right width=100
    border=1 hspace=0 alt=""> 
    </td>
<td>
  <b>RAMP: A Rule-Based Agent for Ms. PacMan</b><br>
  Winner of the 2008 Ms. PacMan competition at the World Congress on
  Computational Intelligence, Hong Kong, May 2008
  <br>
  <small>
    Synopsis: Artificial intelligence approach to game playing.<br>
     Masters project by Alan Fitzgerald<br>
     Contributors: Peter Kemeraitis<br>
     Base code and competetion run by Simon Lucas
     </small>
</td></tr>

</table>



</body>
